cmd/nerdctl/image/image_remove_test.go

```diff
@@ -302,3 +305,53 @@ func TestRemove(t *testing.T) {
 
 	testCase.Run(t)
 }
+
+// TestIssue3016 tests https://github.com/containerd/nerdctl/issues/3016
+func TestIssue3016(t *testing.T) {
+	testCase := nerdtest.Setup()
+
+	const (
+		tagIDKey = "tagID"
+	)
+
+	testCase.SubTests = []*test.Case{
+		{
+			Description: "Issue #3016 - Tags created using the short digest ids of container images cannot be deleted using the nerdctl rmi command.",
+			Setup: func(data test.Data, helpers test.Helpers) {
+				helpers.Ensure("pull", testutil.CommonImage)
+				helpers.Ensure("pull", testutil.NginxAlpineImage)
+
+				img := nerdtest.InspectImage(helpers, testutil.NginxAlpineImage)
+				repoName, _ := imgutil.ParseRepoTag(testutil.NginxAlpineImage)
+				tagID := strings.TrimPrefix(img.RepoDigests[0], repoName+"@sha256:")[0:8]
+
+				helpers.Ensure("tag", testutil.CommonImage, tagID)
+
+				data.Set(tagIDKey, tagID)
+			},
+			Cleanup: func(data test.Data, helpers test.Helpers) {
+				helpers.Anyhow("rmi", "-f", testutil.CommonImage)
```

Comment:
@haytok 

You should not remove these images.
Your test is being run in parallel with other tests that might depend on these images (this is why you are failing docker tests)
It is fine to just let them in.

Reply:
Thanks !!!
Now that I have made sure that the CommonImage is not deleted, let's see if the test succeeds.